workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets

```diff
@@ -49,14 +49,7 @@ Copyright (c) Samsung All rights reserved.
   <ItemGroup>
     <KnownFrameworkReference
       Include="Samsung.Tizen"
-      TargetFramework="net7.0-tizen$(_DefaultTargetPlatformVersion)"
-      TargetingPackName="Samsung.Tizen.Ref"
-      TargetingPackVersion="**FromWorkload**"
-      Profile="Tizen"
-    />
-    <KnownFrameworkReference
-      Include="Samsung.Tizen"
-      TargetFramework="net6.0-tizen$(_DefaultTargetPlatformVersion)"
+      TargetFramework="$(TargetFramework)"
```

Comment:
need to consider the use of implcit tpm version. e.g., `net7.0-tizen`.

Reply:
Done updating :)